CONTRIBUTING.md

```diff
@@ -1,16 +1,89 @@
-# How to contribute to this project
+# Table Of Contents
 
-Please use our tracker for requests for new relations, modifications, clarifications, etc.
+- [Table Of Contents](#table-of-contents)
```

Comment:
There is not really a point the refer to the TOC from the TOC. Remove and reindent the rest?

Reply:
I think it adds automatically because of the title

Reply:
who is "it" in "it automatically"?